.github/workflows/project-codeflare-release.yml

```diff
@@ -22,6 +22,14 @@ on:
       kueue-version:
         description: 'Tested version of Kueue (for example: v0.0.0)'
         required: true
+      cert-generator-image:
+        description: 'Image used to generate RayCluster certificates'
+        required: true
+        default: 'registry.redhat.io/ubi9:latest'
+      oauth-proxy-image:
+        description: 'Image used to run OAuth proxy'
+        required: true
+        default: 'registry.redhat.io/openshift4/ose-oauth-proxy:latest'
```

Comment:
If I remember correctly GitHub doesn't allow more than 10 input parameters for GH action.
Can we use defaults in the build?